src/solvers/dg_simplices/fluxdiff.jl

```diff
@@ -0,0 +1,182 @@
+    du[i] = val_i 
+  end
+end
+
+function build_lazy_physical_derivative(elem::Int, orientation::Int, mesh::VertexMappedMesh{2, Tri}, dg, cache) 
+  @unpack Qrst_skew_Tr = cache
+  @unpack rxJ, sxJ, ryJ, syJ = mesh.md
+  QrskewTr, QsskewTr = Qrst_skew_Tr
+  if orientation == 1
+    return LazyArray(@~ @. 2 * (rxJ[1,elem] * QrskewTr + sxJ[1,elem] * QsskewTr))
+  else
+    return LazyArray(@~ @. 2 * (ryJ[1,elem] * QrskewTr + syJ[1,elem] * QsskewTr))
+  end
+end
+
+function build_lazy_physical_derivative(elem::Int, orientation::Int, mesh::VertexMappedMesh{3, Tet}, dg, cache) 
+  @unpack Qrst_skew_Tr = cache
+  QrskewTr, QsskewTr, QtskewTr = Qrst_skew_Tr
+  @unpack rxJ, sxJ, txJ, ryJ, syJ, tyJ, rzJ, szJ, tzJ = mesh.md
+  if orientation == 1
+    return LazyArray(@~ @. 2 * (rxJ[1,elem]*QrskewTr + sxJ[1,elem]*QsskewTr + txJ[1,elem]*QtskewTr))
+  elseif orientation == 2
+    return LazyArray(@~ @. 2 * (ryJ[1,elem]*QrskewTr + syJ[1,elem]*QsskewTr + tyJ[1,elem]*QtskewTr))
+  elseif orientation == 3
+    return LazyArray(@~ @. 2 * (rzJ[1,elem]*QrskewTr + szJ[1,elem]*QsskewTr + tzJ[1,elem]*QtskewTr))
+  end
+end
+
+function calc_volume_integral!(du, u::StructArray, volume_integral::VolumeIntegralFluxDifferencing,
+                 mesh::VertexMappedMesh, equations, dg::DG, cache) where {DG <: SBPDGFluxDiff}
+
+  rd = dg.basis  
+  @unpack local_values_threaded = cache
+
+  volume_flux_oriented(i) = let i=i, equations=equations 
+    @inline (u_ll, u_rr)->volume_integral.volume_flux(u_ll, u_rr, i, equations)
+  end
+
+  # Todo: simplices. Dispatch on curved/non-curved mesh types, this code only works for affine meshes (accessing rxJ[1,e],...)
+  @threaded for e in eachelement(mesh, dg, cache)
+    rhs_local = local_values_threaded[Threads.threadid()]
+    fill!(rhs_local, zero(eltype(rhs_local)))
+    u_local = view(u, :, e)
+    for i in eachdim(mesh) 
+      Qi_skew_Tr = build_lazy_physical_derivative(e, i, mesh, dg, cache)
+      hadamard_sum_ATr!(rhs_local, Qi_skew_Tr, volume_flux_oriented(i), u_local)
+    end
+    view(du, :, e) .+= rhs_local ./ rd.wq
+  end
+end
+
+
+function create_cache(mesh::VertexMappedMesh{Dim}, equations, dg::DG, 
+                      RealT, uEltype) where {DG <: PolyDGFluxDiff} where {Dim}
+
+  rd = dg.basis
+  @unpack md = mesh
+  
+  # Todo: simplices. Fix this when StartUpDG v0.11.0 releases: new API `Qrst_hybridized, VhP, Ph = StartUpDG.hybridized_SBP_operators(rd)`
+  if Dim==2
+    Qr_hybridized, Qs_hybridized, VhP, Ph = StartUpDG.hybridized_SBP_operators(rd)
+    Qrst_hybridized = (Qr_hybridized, Qs_hybridized) 
+  elseif Dim==3
+    Qr_hybridized, Qs_hybridized, Qt_hybridized, VhP, Ph = StartUpDG.hybridized_SBP_operators(rd)
+    Qrst_hybridized = (Qr_hybridized, Qs_hybridized, Qt_hybridized) 
+  end
+  Qrst_skew_Tr = map(A -> -.5*(A-A'), Qrst_hybridized)
+
+
+  nvars = nvariables(equations)
+
+  # storage for all quadrature points (concatenated volume / face quadrature points)
+  num_quad_pts_total = rd.Nq + rd.Nfq
```

Comment:
I'm not sure whether saving 3 characters is worth the additional mental overhead for people trying to understand the code.

Reply:
Just to check, are you referring to `nvars` or `num_quad_pts_total`? 

If it's `num_quad_pts_total`, how about `num_quad_points_total`?

Reply:
Yes, I was thinking about `num_quad_pts_total` vs. `num_quad_points_total`